src/Backend/Modules/Locale/Layout/Templates/Index.html.twig

```diff
@@ -147,7 +147,7 @@
           <div class="content-title">
             <p>
               {{ 'lbl.Actions'|trans|ucfirst }}
-              <abbr class="fa fa-info-circle" data-toggle="tooltip" title="{{ 'msg.HelpActionValue'|trans }}"></abbr>
+              <abbr class="fa fa-info-circle" aria-hidden="true" data-toggle="tooltip" title="{{ 'msg.HelpActionValue'|trans }}"></abbr>
```

Comment:
Can you undo this? A better fix is available in https://github.com/forkcms/forkcms/pull/2320